scene/resources/texture.cpp

```diff
@@ -1751,6 +1751,7 @@ void AnimatedTexture::set_current_frame(int p_frame) {
 	RWLockWrite r(rw_lock);
 
 	current_frame = p_frame;
+	time = 0
```

Comment:
Thanks for your first contribution. You forgot the semicolon here. After fixing it don't forget to squash commits as it's required by our pipeline (see https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase).